omnibus/omnibus-test.ps1

```diff
@@ -39,5 +38,4 @@ Write-Output "Running verification for $product"
 $Env:GEM_HOME = "C:\SHOULD_NOT_EXIST"
 $Env:GEM_PATH = "C:\SHOULD_NOT_EXIST"
 
-Set-Location -Path C:\opscode\inspec\bin
```

Comment:
What other impact would removing the CWD setting have?

Reply:
I don't see any impact at all in my testing.

I originally copied the following lines from [ci/verify-inspec.bat](https://github.com/inspec/inspec/blob/master/ci/verify-inspec.bat#L8-L9) and converted the `cd` to powershell's native `Set-Location -Path`.

```
cd C:\opscode\inspec\bin
inspec version
```

I just did some more thorough testing though and realized that the line was never really necessary since `C:\opscode\inspec\bin` is already getting prepended to the `PATH` so I removed it to avoid confusion about its presence.